experiment/autobumper/bumper/bumper.go

```diff
@@ -274,16 +278,19 @@ func updatePullRequestWithLabels(gc github.Client, org, repo, title, body, match
 	return nil
 }
 
-// updateReferences update the references of prow-images and/or testimages
+// updateReferences update the references of prow-images and/or boskos-images and/or testimages
 // in the files in any of "subfolders" of the includeConfigPaths but not in excludeConfigPaths
 // if the file is a yaml file (*.yaml) or extraFiles[file]=true
-func updateReferences(bumpProwImages, bumpTestImages bool, targetVersion string,
+func updateReferences(bumpProwImages, bumpBoskosImages bool, bumpTestImages bool, targetVersion string,
```

Comment:
Please unit test this.

Avoid network and disk access for `upstreamImageVersionResolver`, `updateFile` etc with function pointers and/or interfaces.

Reply:
Thanks for the suggestion! Added unit tests as well.